doc/en/announce/release-2.8.3.rst

```diff
@@ -0,0 +1,56 @@
+pytest-2.8.3: bug fixes
+=======================
+
+pytest is a mature Python testing tool with more than a 1100 tests
+against itself, passing on many different interpreters and platforms.
+This release is supposed to be drop-in compatible to 2.8.1.
+
+See below for the changes and see docs at:
+
+    http://pytest.org
+
+As usual, you can upgrade from pypi via::
+
+    pip install -U pytest
+
+Thanks to all who contributed to this release, among them:
+
+    Bruno Oliveira
+    Demian Brecht
+    Florian Bruhin
```

Comment:
I used `git log --pretty='format:%an' 2.8.1..HEAD | sort | uniq` in the past. I'll add that to `HOWTORELEASE`.